osu.ElasticIndexer/SoloScore.cs

```diff
@@ -15,7 +15,11 @@ namespace osu.ElasticIndexer
     [SuppressMessage("Style", "IDE1006")]
     [ElasticsearchType(IdProperty = nameof(id))]
     [ChunkOn(
-        Query = "s.*, (select pp from solo_scores_performance p where p.score_id = s.id) pp, (select country_acronym from phpbb_users u where u.user_id = s.user_id) country_code from solo_scores s",
+        Query = @"s.*,
```

Comment:
Might be best to change to using `JOIN` to avoid two subqueries to the same table:

```sql
s.*, pp, country_acronym, user_warnings FROM solo_scores s JOIN solo_scores_performance ON s.id = score_id JOIN phpbb_users USING (user_id)
```

```csharp
root@db-master:osu> explain select s.*, pp, country_acronym, user_warnings FROM solo_scores s JOIN solo_scores_performance ON
                 ->  s.id = score_id JOIN phpbb_users USING (user_id) limit 10;
| id | select_type | table                   | partitions | type   | possible_keys                     | key     | key_len | ref                                  | rows     | filtered | Extra  |
|----|-------------|-------------------------|------------|--------|-----------------------------------|---------|---------|--------------------------------------|----------|----------|--------|
| 1  | SIMPLE      | solo_scores_performance | <null>     | ALL    | PRIMARY                           | <null>  | <null>  | <null>                               | 25583513 | 100.0    | <null> |
| 1  | SIMPLE      | s                       | <null>     | eq_ref | PRIMARY,solo_scores_user_id_index | PRIMARY | 8       | osu.solo_scores_performance.score_id | 1        | 100.0    | <null> |
| 1  | SIMPLE      | phpbb_users             | <null>     | eq_ref | PRIMARY,country_lookup            | PRIMARY | 4       | osu.s.user_id                        | 1        | 100.0    | <null> |

3 rows in set
Time: 0.117s
root@db-master:osu> explain select s.*,
                 ->          (select pp from solo_scores_performance p where p.score_id = s.id) pp,
                 ->          (select country_acronym from phpbb_users u where u.user_id = s.user_id) country_code,
                 ->          (select user_warnings from phpbb_users u where u.user_id = s.user_id) user_warnings
                 ->          from solo_scores s limit 10;
| id | select_type        | table | partitions | type   | possible_keys          | key     | key_len | ref           | rows     | filtered | Extra  |
|----|--------------------|-------|------------|--------|------------------------|---------|---------|---------------|----------|----------|--------|
| 1  | PRIMARY            | s     | <null>     | ALL    | <null>                 | <null>  | <null>  | <null>        | 32368547 | 100.0    | <null> |
| 4  | DEPENDENT SUBQUERY | u     | <null>     | eq_ref | PRIMARY,country_lookup | PRIMARY | 4       | osu.s.user_id | 1        | 100.0    | <null> |
| 3  | DEPENDENT SUBQUERY | u     | <null>     | eq_ref | PRIMARY,country_lookup | PRIMARY | 4       | osu.s.user_id | 1        | 100.0    | <null> |
| 2  | DEPENDENT SUBQUERY | p     | <null>     | eq_ref | PRIMARY                | PRIMARY | 8       | osu.s.id      | 1        | 100.0    | <null> |

4 rows in set
Time: 0.118s
```